### PR TITLE
Set edgehubuser uid and use ContainerUser on windows

### PR DESCRIPTION
**Pull request by ancaantochi**
- Set edgehubuser uid to 1000 
- Use ContainerUser on windows arm 32

### CODE REVIEW

File: edge-hub/docker/linux/amd64/Dockerfile [hunk 1]
```diff
@@ -10,7 +10,8 @@ RUN apk update && \
     apk add --no-cache snappy libcap
 
 # Add an unprivileged user account for running Edge Hub
-RUN adduser -Ds /bin/sh edgehubuser 
+ARG EDGEHUBUSER_ID=1000
```
Comment (arsing) [inline edge-hub/docker/linux/amd64/Dockerfile]: Does this actually help? The container user namespace will likely always have 1000 available. I believe the issue is not about the container user, but the mapped uid on the host.

Comment (varunpuranik) [inline edge-hub/docker/linux/amd64/Dockerfile]: Why is this not needed for linux/arm?

Comment (arsing) [reply edge-hub/docker/linux/amd64/Dockerfile]: As discussed, seems like Docker *does* have a one-to-one mapping between host and container uids.

Comment (ancaantochi) [reply edge-hub/docker/linux/amd64/Dockerfile]: It is, Angelo build the base images so he also took the change for the uid

**Pull request merged**